.github/workflows/linux.yml

```diff
@@ -1,8 +1,5 @@
 name: Tests on Linux (Ubuntu 20.04, Python 3.11)
 on:
-  schedule:
-    # at 00:00 on Wednesday and Saturday
-    - cron: '0 0 * * 3,6'
```

Comment:
It is used for the `TensorFlow_Hub_Models_Tests` & `PyTorch_Models_Tests` stages to run full validation. Do you want to disable it?

Reply:
reverted